sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSubForSocketIO/src/Services/WebPubSubForSocketIOService.cs

```diff
@@ -32,14 +30,18 @@ public WebPubSubForSocketIOService(Uri endpoint, AzureKeyCredential keyCredentia
         public WebPubSubForSocketIOService(Uri endpoint, TokenCredential credential, string hub)
         {
             _client = new WebPubSubServiceClient(endpoint, hub, credential);
+            _endpoint = endpoint;
             _hub = hub;
             _useConnectionStrings = false;
         }
 
         // For tests.
-        public WebPubSubForSocketIOService(WebPubSubServiceClient client)
+        public WebPubSubForSocketIOService(WebPubSubServiceClient client, Uri endpoint = null, string hub = null, bool useConnectionStrings = false)
         {
             _client = client;
```

Comment:
we also have client.Endpoint in WebPubSubServiceClient, endpoint parameter is not needed?

Reply:
That's correct